scripts/plot_dataframe.py

```diff
@@ -55,7 +55,7 @@
 def load_trades(args: Namespace, pair: str, timerange: TimeRange) -> pd.DataFrame:
     trades: pd.DataFrame = pd.DataFrame()
     if args.db_url:
-        persistence.init(args.db_url, True)
+        persistence.init(args.db_url, False)
```

Comment:
minor: imho, that will be more readable to use the name of the second parameter here:
i.e. `persistence.init(args.db_url, clean_open_orders=False)`

rest LGTM 👍